pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -274,13 +274,13 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env model.Env
 				direction:        http_conn.INGRESS,
 			}
 			if listenerOpts.tlsMultiplexed {
+				// If multiplexed is needed, setup to filter chain match, one for "istio" ALPN TLS, one for default.
```

Comment:
This is describing implementation and not the feature. I'd suggest that you replace it with something like: `Allow both: istio mTLS ("istio" ALPN) and legacy clients.`

Reply:
Done.